### DISCUSSION TITLE
Send mail

### DISCUSSION THREAD

**Discussion by letienhai02** [inbucket/inbucket#295] [Q&A] [answered]
I have a java spring boot project. how to send mail and receive confirmation on inbucket?
I don't quite understand how it works. Pls help me.

**Comment by jhillyerd** [comment, marked as answer]
I've never used spring boot, but I think you could follow https://mkyong.com/spring-boot/spring-boot-how-to-send-email-via-smtp/ with a configuration similar to

```
spring.mail.host=<name or IP of your inbucket install>
spring.mail.port=2500
spring.mail.properties.mail.smtp.auth=false
spring.mail.properties.mail.smtp.connectiontimeout=5000
spring.mail.properties.mail.smtp.timeout=5000
spring.mail.properties.mail.smtp.writetimeout=5000
```

**Reply by letienhai02** [reply]
Thank you for support. 
I have misconfigured port. 
After changing it to 2500, I was able to connect.
